src/Helpers/BasicAuth.php

```diff
@@ -0,0 +1,84 @@
+<?php
+namespace Dumbo\Helpers;
+
+use Dumbo\Context;
+
+class BasicAuth
+{
+    /**
+     * Create a middleware that checks for Basic Auth credentials in the Authorization header
+     *
+     * @param array $options The options for the basic authentication middleware
+     * @return callable The middleware
```

Comment:
```suggestion
     * @return callable The middleware
     * @throws \InvalidArgumentException If the options are invalid
```

It might be worth implementing `InvalidAurgumentException` so it's clear this will throw.